avt_vimba_camera/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.5)
 project(avt_vimba_camera)
 
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
why change it into c++17?

Reply:
More useful function in STL, however, I can downgrade this to 14.

Reply:
Ok, so I will revert this to c++14 today

Reply:
@icolwell-as @jilinzhouas Ok, code reverted to C++14.